mxd/backend-service/src/main/java/org/eclipse/mxd/util/DateTimeUtil.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *    Contributors:Ravinder Kumar
+ *    Backend-API and implementation
+ *
+ ******************************************************************************/
+
+package org.eclipse.mxd.util;
+
+import java.util.Date;
+
+public class DateTimeUtil {
+
+    public static Date getCurrentDateTimeInAsiaKolkata() {
```

Comment:
Date class uses system timezone, so if server is in some other location, time will be in that zone. Hence method name is not suitable. You can use `new Date()` directly instead of this method. Also `Date` class has lots of issue. Check, if you could use Instant class.

Reply:
System time zone is removed and directly used new Date()